dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterExecThread.java

```diff
@@ -651,14 +653,23 @@ private void setTaskNodeSkip(List<String> taskNodesSkipList){
      * submit post node
      * @param parentNodeName parent node name
      */
+    private Map<String,Object> propToValue = new ConcurrentHashMap<String, Object>();
     private void submitPostNode(String parentNodeName){
 
         List<String> submitTaskNodeList = parsePostNodeList(parentNodeName);
 
         List<TaskInstance> taskInstances = new ArrayList<>();
         for(String taskNode : submitTaskNodeList){
+            try {
+                VarPoolUtils.convertVarPoolToMap(propToValue, processInstance.getVarPool());
+            } catch (ParseException e) {
+                logger.error("parse {} exception", processInstance.getVarPool(), e);
+                throw new RuntimeException();
+            }
+            TaskNode taskNodeObject = dag.getNode(taskNode);
+            VarPoolUtils.setTaskNodeLocalParams(taskNodeObject, propToValue);
```

Comment:
there is bug will cause the origin param be replace by the task param,so the task will get nothing when use  taskExecutionContext.getTaskParams()